.eslintrc.js

```diff
@@ -213,7 +213,10 @@ module.exports = {
     'one-var': ['error', { initialized: 'never' }],
     'one-var-declaration-per-line': 'error',
     'operator-linebreak': ['error', 'after'],
-    'prefer-const': ['error', { ignoreReadBeforeAssign: true }],
+    'prefer-const': ['error', {
+      ignoreReadBeforeAssign: true,
+      destructuring: 'all',
```

Comment:
This relaxing of the lint rule seems to be to accommodate just a single line in js2c.js. Instead of relaxing the lint rule everywhere, can we change line 178 of the file to disable the rule for just that line?:

```js
for (let { source, input } of queue) { // eslint-disable-line prefer-const
```

(The comment at the end of the line will disable the `prefer-const` rule for that line only.)

Reply:
BTW don't `for ... of` loops work with `const`?